multinet/api.py

```diff
@@ -24,11 +27,62 @@
     AlreadyExists,
     RequiredParamsMissing,
 )
-from multinet.user import current_user
+from multinet.user import current_user, find_user_from_id
 
 bp = Blueprint("multinet", __name__)
 
 
+# Included here due to circular imports
+# TODO: Remove once implementing new ORM and permission storage
+def _permissions_id_to_user(permissons: WorkspacePermissions) -> Dict:
+    """Transform permission documents to directly container user info."""
+
+    new_permissions = deepcopy(permissons)
+    for role, users in new_permissions.items():
+        if role == "public":
+            continue
+
+        if role == "owner":
+            assert isinstance(users, str)
+
+            user = find_user_from_id(users)
+            if user is not None:
+                # Not sure why mypy complains about `user` here
```

Comment:
What is the exact type error you get?

Reply:
It was giving me an error regarding user being a `str` or something. However I'm actually about to push a change which will fix the type errors, so this will be moot.